### PR TITLE
[TestMigration] Migration of activate/deactivate snap test.

### PR DESCRIPTION
**Pull request by srijan-sivakumar**
The test case in glusto being,
    [Test](https://github.com/gluster/glusto-tests/blob/master/tests/functional/snapshot/test_activate_deactivate.py)
    
    Updates: #292
    
    Signed-off-by: srijan-sivakumar <ssivakum@redhat.com>

### CODE REVIEW

**Comment by srijan-sivakumar** [comment]
```console
Results for snapshot
Test nature : disruptive
TestName : test_activate_deactivate
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     rep     |     PASS    |        00:01:12       |      NA     |
|     dist    |     PASS    |        00:01:09       |      NA     |
|   dist-rep  |     PASS    |        00:01:42       |      NA     |
|     disp    |     PASS    |        00:01:20       |      NA     |
|  dist-disp  |     PASS    |        00:01:58       |      NA     |
+-------------+-------------+-----------------------+-------------+
+--------------------------------------+-------+
|               Category               | Count |
+--------------------------------------+-------+
|           Disruptive Tests           |   1   |
|         Non Disruptive Tests         |   0   |
|       Disruptive Tests Skipped       |   0   |
|     Non Disruptive Tests Skipped     |   0   |
|              Tests Ran               |   1   |
|            Tests Skipped             |   0   |
|             Total Tests              |   1   |
|   Disruptive Tests Pass Percentage   | 100.0 |
| Non Disruptive Tests Pass Percentage |   0   |
|        Total Pass Percentage         | 100.0 |
+--------------------------------------+-------+
Total Time Taken : 00 hours 07 minutes 46 seconds
```

File: tests/functional/snapshot/test_activate_deactivate.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along
+with this program; if not, write to the Free Software Foundation, Inc.,
+51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+"""
```
Comment (nik-redhat) [inline tests/functional/snapshot/test_activate_deactivate.py:17]: nit: description

File: tests/functional/snapshot/test_activate_deactivate.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+from tests.d_parent_test import DParentTest
+
+# disruptive;rep,dist,dist-rep,disp,dist-disp
+
+
+class TestCase(DParentTest):
+
+    def run_test(self, redant):
+        # pylint: disable=too-many-branches, too-many-statements
```
Comment (nik-redhat) [inline tests/functional/snapshot/test_activate_deactivate.py:28]: I think we don't either of these pylint considerations

**Review by nik-redhat — COMMENTED** [review]
Two nit picks, else we are good.

**Review by aujjwal-redhat — COMMENTED** [review]
LGTM. jus the nits as @nik-redhat mentioned

File: tests/functional/snapshot/test_activate_deactivate.py (hunk 1 above)
Comment (srijan-sivakumar) [reply tests/functional/snapshot/test_activate_deactivate.py:21]: Ack.

File: tests/functional/snapshot/test_activate_deactivate.py (hunk 2 above)
Comment (srijan-sivakumar) [reply tests/functional/snapshot/test_activate_deactivate.py]: Ack.

**Review by aujjwal-redhat — APPROVED** [review]

**Review by nik-redhat — APPROVED** [review]

**Pull request merged**